docs/src/methods/fourier_surrogates.md

```diff
@@ -43,10 +48,49 @@ s = surrogate(ts, PartialRandomization(0.5))
 surroplot(ts, s)
 ```
 
+In addition to [`PartialRandomization`](@ref), we provide two other algorithms for producing partially randomized surrogates, outlined below.
+
+### Relative partial randomization
+
+The [`PartialRandomization`](@ref) algorithm corresponds to assigning entirely new phases to the Fourier spectrum with some degree of randomness, regardless of any deterministic structure in the original phases. As such, even for $\alpha = 0$ the surrogate time series can differ drastically from the original time series.
+
+By contrast, the [`RelativePartialRandomization`](@ref) procedure draws phases as:
+
+$$\phi \to \phi + \alpha \xi, \quad \xi \sim \mathcal{U}(0, 2\pi).$$
+
+With this algorithm, phases are progressively corrupted by higher values of $\alpha$: surrogates are identical to the original time series for $\alpha = 0$, equivalent to random noise for $\alpha = 1$, and retain some of the structure of the original time series when $0 < \alpha < 1$. This procedure is particularly useful for controlling the degree of chaoticity and non-linearity in surrogates of chaotic systems.
+
+### Spectral partial randomization
+
+Both of the algorithms above randomize phases at all frequency components to the same degree.
+To assess the contribution of different frequency components to the structure of a time series, the [`SpectralPartialRandomization`](@ref) algorithm only randomizes phases above a frequency threshold.
+The threshold is chosen as the lowest frequency at which the power spectrum of the original time series drops below a fraction $1-\alpha$ of its maximum value (such that the power contained above the frequency threshold is a proportion $\alpha$ of the total power, excluding the zero frequency).
+
+See the figure below for a comparison of the three partial randomization algorithms:
+```@example MAIN
+using DynamicalSystemsBase # hide
+@inbounds function lorenz_rule!(du, u, p, t) # hide
+    σ = p[1]; ρ = p[2]; β = p[3] # hide
+    du[1] = σ*(u[2]-u[1]) # hide
+    du[2] = u[1]*(ρ-u[3]) - u[2] # hide
+    du[3] = u[1]*u[2] - β*u[3] # hide
+    return nothing # hide
+end # hide
+u0 = [0, 10.0, 0] # hide
+p0 = [10, 28, 8/3] # hide 
+diffeq = (; abstol = 1e-9, reltol = 1e-9) # hide 
+lorenz = CoupledODEs(lorenz_rule!, u0, p0; diffeq) # hide
+x = trajectory(lorenz, 1000; Ttr=500, Δt=0.025)[1][:, 1] # hide
+x = TimeseriesSurrogates.DelayEmbeddings.standardize(x) # hide
+A = [PartialRandomization, RelativePartialRandomization, SpectralPartialRandomization] # hide
```

Comment:
@brendanjohnharris Since you've now updated the docstring of the `surrocompare` function, I think it would be nice if the variable names in the example reflected that too. Perhaps rename `A` to `surr_types`?